Containerfile

```
@@ -1,47 +1,64 @@
-#syntax=docker/dockerfile-upstream:1.4.0-rc1
-ARG RUSTUP_TOOLCHAIN
-FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-build
-RUN apt-get update &&\
-    apt-get install -y curl
+#syntax=docker/dockerfile-upstream:1.4
+
 
-RUN cargo install cargo-chef
+# Base image for builds and cache
+ARG RUSTUP_TOOLCHAIN
+FROM rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-build
```

Perhaps we should keep this? For reference: https://github.com/shuttle-hq/shuttle/pull/723
```suggestion
FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-build
```